cmd/fireeth/main.go

```diff
@@ -94,9 +96,14 @@ func Chain() *firecore.Chain[*pbeth.Block] {
 				return nil, fmt.Errorf("rpc endpoints cannot contain commas")
 			}
 
+			if balanceEndpoint != "" && strings.Contains(balanceEndpoint, ",") {
```

Comment:
we support multiple (retryable) endpoints for eth_calls, why not for get_balance ?